docs/src/IconButton.doc.js

```diff
@@ -22,7 +22,6 @@ card(
       {
         name: 'accessibilityLabel',
         type: 'string',
-        required: true,
```

Comment:
In the types, this is still required. Curious why you removed it

Reply:
Mmmmm....

Reply:
DONE.